src/bodyElement.ts

```diff
@@ -307,20 +313,50 @@ const parse = (context: Context, atoms?: Atoms) =>
                 return new Err('No atom data returned by capi')
             }
 
-            const id = element.contentAtomTypeData?.atomId
-            const atom = atoms.interactives?.find(interactive => interactive.id === id);
+            switch (element.contentAtomTypeData?.atomType) {
+                case "interactive": {
```

Comment:
That feels like a lot to put in that switch statement. Would extracting this into its own function make it more readable?